### DISCUSSION TITLE
Some Alias Suggestions

### DISCUSSION THREAD

**Discussion by KaguraSayuki** [Arcaea-Infinity/BotArcAPIs-Memories#16] [Song Alias] [answered]
New alias requests for these songs:

- Felis (*felis*) - 凤梨酥
- Felis (*felis*) - 菲林
- Oshama Scramble! (*oshamascramble*) - 希腊奶
- Fracture Ray (*fractureray*) - 断裂射线

**Comment by TheSnowfield** [comment, marked as answer]
Thanks!
